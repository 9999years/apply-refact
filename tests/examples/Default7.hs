yes = "test" ++ concatMap (' ':) ["of","this"]
