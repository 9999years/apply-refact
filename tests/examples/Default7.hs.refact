[("tests/examples/Default7.hs:1:7-46: Warning: Use unwords\nFound:\n  \"test\" ++ concatMap (' ' :) [\"of\", \"this\"]\nPerhaps:\n  unwords (\"test\" : [\"of\", \"this\"])\n",[Replace {rtype = Expr, pos = SrcSpan {startLine = 1, startCol = 7, endLine = 1, endCol = 47}, subts = [("x",SrcSpan {startLine = 1, startCol = 7, endLine = 1, endCol = 13}),("y",SrcSpan {startLine = 1, startCol = 34, endLine = 1, endCol = 47})], orig = "unwords (x : y)"}])]
