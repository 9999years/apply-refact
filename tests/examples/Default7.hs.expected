yes = unwords ("test" : ["of","this"])
